app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -324,7 +324,9 @@ module.exports = {
       candidate.delegator_shares = (
         parseInt(candidate.delegator_shares) + amount
       ).toString()
-      storeTx("cosmos-sdk/MsgDelegate", tx)
+
+      // storeTx("cosmos-sdk/MsgDelegate", tx) // do we need this anymore?
```

Comment:
yes, this stores the txs in the txs list to be able to show them in the transaction history